winfetch.ps1

```diff
@@ -644,11 +644,13 @@ function info_disk {
             if ($diskInfo.DeviceID -eq $diskLetter -or $diskLetter -eq "*") {
                 $total = $diskInfo.Size
                 $used = $total - $diskInfo.FreeSpace
-                $usage = [math]::floor(($used / $total * 100))
-                [void]$lines.Add(@{
-                    title   = "Disk ($($diskInfo.DeviceID))"
-                    content = get_level_info "" $diskstyle $usage "$(to_units $used) / $(to_units $total)"
-                })
+		if ($total -gt 0) {
+		    $usage = [math]::floor(($used / $total * 100))
+		    [void]$lines.Add(@{
+		    title   = "Disk ($($diskInfo.DeviceID))"
+		    content = get_level_info "" $diskstyle $usage "$(to_units $used) / $(to_units $total)"
```

Comment:
Can you indent these two lines (650, 651)

Reply:
yes, as you can see from these 2 screenshots the discs space shows up just fine when there is a disc in the drive
![image](https://user-images.githubusercontent.com/75815262/114108024-0b265300-9887-11eb-90df-b901e4c700c5.png)

![image](https://user-images.githubusercontent.com/75815262/114108078-2b561200-9887-11eb-8541-6d1cef29c8d7.png)